tools/webcam/camerad.py

```diff
@@ -26,7 +26,6 @@ def __init__(self):
     self.cameras = []
     for c in CAMERAS:
       cam = Camera(c.msg_name, c.stream_type, c.cam_id)
-      assert cam.cap.isOpened(), f"Can't find camera {c}"
```

Comment:
Changing the above to `def __init__(self) -> None:` would allow mypy to catch this btw. We have too many errors, so we only run mypy on explicitly typed functions, I would like to change that eventually.

I think pylint might have caught these missing attribute calls as well, waiting on ruff to implement that check!